guide/glossary/address.md

```diff
@@ -89,24 +89,10 @@ When the output of a transaction is used as the input of another transaction, it
 
 Since "legacy" addresses are still in use, some older applications have yet to upgrade. Script addresses can be used to resolve incompatibility issues that can arise when the sender's application does not recognize the SegWit address provided by the receiver.
 
-<div class="center" markdown="1">
```

Comment:
What's the reason for removing this image?

Reply:
Especially since it's in the glossary and this PR is about the requesting page.

Reply:
It was deleted prior just removed the markdown for it, will update it and re-add it. I don't think we really need images in this section though tbh - feels out of place.